config/common/server.yaml

```diff
@@ -0,0 +1,267 @@
+{{ range $index, $Map := .ServerCount }}
+persistentVolumeClaims:
+  - metadata:
+      name: "{{.ApplicationName}}-postgresql-claim-{{$index}}"
+      labels:
+        app: "{{.ApplicationName}}"
+        service: "{{.ApplicationName}}-postgresql-{{$index}}"
+    spec:
+      accessModes:
+        - ReadWriteOnce
+      resources:
+        requests:
+          storage: "1Gi"
+## PostgreSQL persistent volume claim END
+## KIE server deployment config BEGIN
+deploymentConfigs:
+  - metadata:
+      name: "{{.ApplicationName}}-kieserver-{{$index}}"
+      labels:
+        app: "{{.ApplicationName}}"
+    spec:
+      strategy:
+        type: Recreate
+      triggers:
+        - type: ImageChange
+          imageChangeParams:
+            automatic: true
+            containerNames:
+              - "{{.ApplicationName}}-kieserver"
+            from:
+              kind: ImageStreamTag
+              namespace: "openshift"
+              name: "rhpam{{.Version}}-kieserver-openshift:{{.ImageTag}}"
+        - type: ConfigChange
+      replicas: 3
+      selector:
+        deploymentConfig: "{{.ApplicationName}}-kieserver-{{$index}}"
+      template:
+        metadata:
+          name: "{{.ApplicationName}}-kieserverblh-{{$index}}"
```

Comment:
appears there's some extra characters in this name? `blh` ?

Reply:
This was actually meant to prove that the name is overwritten